src/components/learnDetail/memo/MemoList.tsx

```diff
@@ -1,20 +1,42 @@
-import { MemoData } from '@src/services/api/types/learn-detail';
+import { HighlightData } from '@src/services/api/types/learn-detail';
 import { COLOR } from '@src/styles/color';
+import { useEffect, useState } from 'react';
 import styled from 'styled-components';
 import Memo from './Memo';
 
 interface MemoListProps {
-  memoList: MemoData[];
+  highlightList: HighlightData[];
+  newMemo: boolean;
```

Comment:
```suggestion
  isNewMemo: boolean;
```
boolean 값이니까 is~~로 수정합시당 !!